actions/software.mk

```diff
@@ -27,7 +27,7 @@ include ../../../software/config.mk
 CFLAGS += -std=c99 -I../../../software/include
 DESTDIR ?= /usr
 libs += ../../../software/lib/libsnap.a
-LDLIBS += $(libs) -lpthread
+LDLIBS += $(libs) -lpthread -lm
```

Comment:
ok following your recommendation, it is now moved to the specific app Makefile, and it works ok. Thx